src/radians.rs

```diff
@@ -34,6 +34,11 @@ impl Radians {
     pub fn value(self) -> f64 {
         self.value
     }
+
+    /// Convert degrees to radians
+    pub fn from_degrees(degrees: f64) -> Result<Self, RadiansError> {
```

Comment:
After internal discussion, me and @bash settled on `try_from_degrees`

Reply:
Resolved in 1a875aa90ffead22e502431154fdccd7e47ab25a